apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -134,24 +153,29 @@ export default function SectionsSetUpContainer() {
           <AdvancedSettingToggles
             updateSection={(key, val) => updateSection(0, key, val)}
             section={sections[0]}
-            assignedUnitTextToSpeechEnabled={true}
-            assignedUnitLessonExtrasAvailable={true}
+            hasLessonExtras={sections[0].course.hasLessonExtras}
+            hasTextToSpeech={sections[0].course.hasTextToSpeech}
             label={i18n.pairProgramming()}
           />
         )}
       </div>
       <div className={moduleStyles.buttonsContainer}>
+        {isNewSection && (
+          <Button
+            icon="plus"
+            text={i18n.addAnotherClassSection()}
+            color="white"
+            onClick={e => {
+              e.preventDefault();
+              console.log('Add Another Class Section clicked');
+            }}
+          />
+        )}
+        {/* TO DO: currently this button just changes text if it is a "new" or "editied"
+        screen, depending on how we want the functionality of this button to work,
+        this might mean creating a different button for the "edit" page */}
         <Button
-          icon="plus"
```

Comment:
Oh nice! Looks like you're tackling https://codedotorg.atlassian.net/browse/TEACH-421 right here. Any chance you can link this ticket: https://codedotorg.atlassian.net/browse/TEACH-278 and this pr so we can find what the button should look like when we're reenabling?

Reply:
Will do!  I already got TEACH-421.  I did comment this out in the newest commit.  I will note it int TEACH-278 too.

Reply:
Yesss thanks!